fast_despawn/data/fast_despawn/functions/loop.mcfunction

```diff
@@ -2,12 +2,14 @@ execute if score Kelp despawnItems = #ONE despawnItems run execute as @e[nbt={It
 execute if score Dirt despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:dirt"}}] run data merge entity @s {Age:5800s}
 execute if score Stone despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:stone"}}] run data merge entity @s {Age:5800s}
 execute if score Stone despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:cobblestone"}}] run data merge entity @s {Age:5800s}
-execute if score Gravel despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:gravel"}}] run data merge entity @s {Age:5900s}
+execute if score Gravel despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:gravel"}}] run data merge entity @s {Age:5800s}
 execute if score Netherrack despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:ender_pearl"}}] run data merge entity @s {Age:5800s}
 execute if score EnderPearl despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:netherrack"}}] run data merge entity @s {Age:5800s}
-execute if score SpecialStone despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:granite"}}] run data merge entity @s {Age:5800s}
-execute if score SpecialStone despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:andesite"}}] run data merge entity @s {Age:5800s}
-execute if score SpecialStone despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:diorite"}}] run data merge entity @s {Age:5800s}
+execute if score StoneVariants despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:granite"}}] run data merge entity @s {Age:5800s}
+execute if score StoneVariants despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:andesite"}}] run data merge entity @s {Age:5800s}
+execute if score StoneVariants despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:diorite"}}] run data merge entity @s {Age:5800s}
+execute if score Grass despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:grass"}}] run data merge entity @s {Age:5800s}
+execute if score Sand despawnItems = #ONE despawnItems run execute as @e[nbt={Item:{id:"minecraft:sand"}}] run data merge entity @s {Age:5800s}
```

Comment:
The `#ONE` constant isn't needed, you can just `execute if score X despawnItems matches 1`, mostly mentioning cause the `#ONE` constant was suggested to be removed for an `#INITIALIZED` one or something along the lines